client/src/components/NavBar.jsx

```diff
@@ -1,16 +1,22 @@
 import {
-  Link
+    Link, useLocation
 } from "react-router-dom";
+import React from "react";
+import "./NavBar.css"
+
 export default function NavBar() {
 
-  return (
-    <div>
-      <Link to='/Followers'>
-        Followers
+    const location = useLocation();
+    return (
+        <div className='MainBar'>
+            <Link className={location.pathname === '/Followers' ? 'NavBarItem theCurrent' : 'NavBarItem'}
```

Comment:
Maybe we can try <Link className={`NavBarItem ${location.pathname === '/Followers' ? 'theCurrent' : ''}`} to prevent repeating "NavBarItem"

Reply:
Both are fine :+1:

Reply:
Thanks for tip hocams 👍